README.md

```diff
@@ -225,7 +225,6 @@ Further implementations are on the way:
 
 - [In progress](https://github.com/microsoft/TypeScript/pull/30829) in TypeScript
 - [Out for review](https://bugs.webkit.org/show_bug.cgi?id=174212) in Safari/JSC
-- [In progress](https://bugzilla.mozilla.org/show_bug.cgi?id=1499448) in Firefox/SpiderMonkey
```

Comment:
Maybe we can add a link above like others which refers to the version the feature released in?